web/src/main/java/it/pagopa/selfcare/mscore/web/controller/DelegationController.java

```diff
@@ -108,4 +108,24 @@ public ResponseEntity<List<DelegationResponse>> getDelegations(@ApiParam("${swag
                 .map(delegationMapper::toDelegationResponse)
                 .collect(Collectors.toList()));
     }
+
+    /**
+     * The function delete delegation
```

Comment:
you could also add that the function setting delegation active to false on instutiton if it has not delegations anymore